### ISSUE TITLE
Add channel_Image.cfg to examples

### ISSUE THREAD

**Issue by pllim** [ejeschke/ginga#171]
Add `channel_Image.cfg` to `examples/config`.

**Comment by ejeschke** [comment]
Hmm.  Not opposed to this, but what would we put in it?  It's basically a dump of the `Preferences` for the generic, starting channel.  As such, it could be configured a lot of ways, and will get overwritten if the user ever saves preferences for that channel.

Are you thinking just to document what settings affect what things?

**Comment by pllim** [comment]
Yes, it's more for documentation. I opened up my `channel_Image.cfg` and not sure what is what for some of the entries.

**Comment by pllim** [comment]
Here is a dump out when I save my channel preference (I think they should mostly be default values):
```python
auto_orient = True
autocenter = 'on'
autocut_method = 'zscale'
autocut_params = {}
autocuts = 'override'
autozoom = 'on'
color_algorithm = 'linear'
color_hashsize = 65535
color_map = 'gray'
cuts = (0.0, 0.0)
defer_lagtime = 0.025
defer_redraw = True
flip_x = False
flip_y = False
follow_focus = True
genthumb = True
icc_black_point_compensation = False
icc_output_intent = 'perceptual'
icc_output_profile = None
icc_proof_intent = 'perceptual'
icc_proof_profile = None
image_overlays = True
intensity_map = 'ramp'
interpolation = 'basic'
numImages = 10
onscreen_ff = 'Sans Serif'
output_icc = None
pan = (1.0, 1.0)
pan_coord = 'data'
profile_use_cuts = False
profile_use_pan = False
profile_use_rotation = False
profile_use_scale = False
profile_use_transform = False
raisenew = True
rot_deg = 0.0
scale = (1.0, 1.0)
scale_max = 10000.0
scale_min = 1e-05
scale_x_base = 1.0
scale_y_base = 1.0
show_pan_position = False
swap_xy = False
switchnew = True
wcs_coords = 'icrs'
wcs_display = 'sexagesimal'
zoom_algorithm = 'step'
zoom_rate = 1.4142135623730951
zoomlevel = 1.0
```

Would be nice to:

1. Group them into relevant categories and insert useful comments.
2. Mention which ones can also be inherited from `general.cfg` (e.g., `numImages`).

**Comment by pllim** [comment]
Resolved by #211.
